TimeTable/Categories/WorkTime/WorkTimeController.swift

```diff
@@ -126,12 +127,44 @@ extension WorkTimeController: UIPickerViewDataSource {
     }
 }
 
+// MARK: -
+extension WorkTimeController: UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: TagCollectionViewCell.reuseIdentifier,
+                                                            for: indexPath) as? TagCollectionViewCell else {
+                                                                return UICollectionViewCell()
+        }
+        guard let tag = self.viewModel.viewRequestedForTag(at: indexPath) else { return UICollectionViewCell() }
+        let isSelected = self.viewModel.viewRequestIfTagIsSelected(at: indexPath)
```

Comment:
Stick to one naming strategy in one commit - `viewDidRequest...` or `viewRequested...` do not mix it like that